src/main/java/net/snowflake/client/core/SFArrowResultSet.java

```diff
@@ -33,45 +33,32 @@
 /** Arrow result set implementation */
 public class SFArrowResultSet extends SFBaseResultSet implements DataConversionContext {
   static final SFLogger logger = SFLoggerFactory.getLogger(SFArrowResultSet.class);
-
+  /** is array bind supported */
+  private final boolean arrayBindSupported;
+  /** total chunk count, not include first chunk */
+  private final long chunkCount;
+  /** time when first chunk arrived */
+  private final long firstChunkTime;
+  /** telemetry client to push stats to server */
+  private final Telemetry telemetryClient;
+  /** statement generate current result set */
+  protected SFStatement statement;
   /** iterator over current arrow result chunk */
   private ArrowChunkIterator currentChunkIterator;
-
```

Comment:
Changes here are a result of running `jfmt` (google-java-format), apologies. No substantive change here other than reformatting.